lib/token.h

```diff
@@ -82,7 +82,7 @@ struct TokenImpl {
     nonneg int mIndex{};
 
     /** Bitfield bit count. */
-    unsigned char mBits{};
+    char mBits = -1;
```

Comment:
looks weird imho to use `char` for a integral value. please use `int8_t` instead.